CHANGELOG.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 
 master
 ------
+- (`#49 <https://github.com/znicholls/silicone/pull/49>`_) Rewrote the documentation and notebooks to update, split up information and clarify.
```

Comment:
```suggestion
- (`#49 <https://github.com/znicholls/silicone/pull/49>`_) Rewrote the documentation and notebooks to update, split up and clarify information.
```

Or have I misread?

Reply:
clarify originally referred to the notebooks in general, but if that's not clear it can be changed

Reply:
ah ok all good let's leave it then